zerver/webhooks/grafana/view.py

```diff
@@ -33,8 +35,18 @@ def api_grafana_webhook(
     if payload.get("message") is not None:
         message_text = payload["message"] + "\n\n"
 
+    alert_status = ""
+    if payload.get("state") is not None:
+        if payload.get("state") == "alerting":
+            alert_status = ":alert: " + "**" + payload["state"].capitalize() + "**" + "\n\n"
+        elif payload.get("state") == "ok":
+            alert_status = ":squared_ok: " + "**" + payload["state"].capitalize() + "**" + "\n\n"
+        else:
+            alert_status = "**" + payload["state"].capitalize() + "**" + "\n\n"
+
```

Comment:
So there has been no tests written or edited for this so tests will fail. You will need to update `zerver/webhooks/grafana/tests.py` and you can then run these with `./tools/test-backend zerver/webhooks/grafana` and all need to be successful.

You can read more about testing here - https://zulip.readthedocs.io/en/latest/testing/testing.html